docs/relational-databases/system-functions/sys-fn-cdc-increment-lsn-transact-sql.md

```diff
@@ -64,7 +64,26 @@ SET @to_lsn = sys.fn_cdc_get_max_lsn();
 SELECT * from cdc.fn_cdc_get_all_changes_HumanResources_Employee( @from_lsn, @to_lsn, 'all' );  
 GO  
 ```  
-  
+**NOTE:**
+
+ Error 313 is expected if LSN range supplied is not appropriate when calling cdc.fn_cdc_get_all_changes_<capture_instance> or cdc.fn_cdc_get_net_changes_<capture_instance> and it should be handled. This example shows how to handle this scenario for dbo.HR_Department table.
+
+```
+begin try
+DECLARE @from_lsn binary(10), @to_lsn binary(10), @save_to_lsn binary(10); 
+SET @save_to_lsn = sys.fn_cdc_get_min_lsn('dbo_HR_Department')
+select @save_to_lsn
+SET @from_lsn = sys.fn_cdc_increment_lsn(0x00000026000005F00003); 
+select @from_lsn
+SET @to_lsn = sys.fn_cdc_get_max_lsn(); 
+select @to_lsn
+SELECT * from cdc.fn_cdc_get_all_changes_dbo_Tab1( @from_lsn, @to_lsn, 'all' ); 
+end try
+begin catch
+if @@ERROR = 313
+RAISERROR ('Invalid range passed to fn_cdc_get_all_changes_dbo_HR_Department.', 16, 1); 
+```
+

```

Comment:
```suggestion
### Note on Error 313 

Error 313 is expected if LSN range supplied is not appropriate when calling `cdc.fn_cdc_get_all_changes_<capture_instance>` or `cdc.fn_cdc_get_net_changes_<capture_instance>` and it should be handled. This example shows how to handle this scenario for a fictional `dbo.HR_Department` table.

```
BEGIN TRY
	DECLARE @from_lsn binary(10), @to_lsn binary(10), @save_to_lsn binary(10); 
	SET @save_to_lsn = sys.fn_cdc_get_min_lsn('dbo_HR_Department')
	SELECT @save_to_lsn
	SET @from_lsn = sys.fn_cdc_increment_lsn(0x00000026000005F00003); 
	SELECT @from_lsn
	SET @to_lsn = sys.fn_cdc_get_max_lsn(); 
	SELECT @to_lsn
	SELECT * from cdc.fn_cdc_get_all_changes_dbo_Tab1( @from_lsn, @to_lsn, 'all' ); 
END TRY
BEGIN CATCH
	IF @@ERROR = 313
	BEGIN
		RAISERROR ('Invalid range passed to fn_cdc_get_all_changes_dbo_HR_Department.', 16, 1); 
	END
END CATCH
```

```